test/local/routes/totp.js

```diff
@@ -7,12 +7,13 @@
 const assert = require('insist')
 const getRoute = require('../../routes_helpers').getRoute
 const mocks = require('../../mocks')
+const otplib = require('otplib')
 const P = require('../../../lib/promise')
 const sinon = require('sinon')
-const proxyquire = require('proxyquire').noPreserveCache()
```

Comment:
Benefit from https://github.com/mozilla/fxa-auth-server/pull/2371/files#r178127428, is that we no longer need proxyrequire.